OpenRA.Game/Graphics/Animation.cs

```diff
@@ -50,18 +50,22 @@ public Animation(World world, string name, Func<WAngle> facingFunc, Func<bool> p
 		}
 
 		public int CurrentFrame => backwards ? CurrentSequence.Length - frame - 1 : frame;
+
 		public Sprite Image => CurrentSequence.GetSprite(CurrentFrame, facingFunc());
 
 		public IRenderable[] Render(WPos pos, in WVec offset, int zOffset, PaletteReference palette)
 		{
 			var tintModifiers = CurrentSequence.IgnoreWorldTint ? TintModifiers.IgnoreWorldTint : TintModifiers.None;
 			var alpha = CurrentSequence.GetAlpha(CurrentFrame);
-			var imageRenderable = new SpriteRenderable(Image, pos, offset, CurrentSequence.ZOffset + zOffset, palette, CurrentSequence.Scale, alpha, float3.Ones, tintModifiers, IsDecoration);
+			var (image, rotation) = CurrentSequence.GetSpriteWithRotation(CurrentFrame, facingFunc());
+			var imageRenderable = new SpriteRenderable(image, pos, offset, CurrentSequence.ZOffset + zOffset, palette,
+													   CurrentSequence.Scale, alpha, float3.Ones, tintModifiers, IsDecoration, rotation);
```

Comment:
Since @penev92 is now the second person complaining about line breaks, can we change these back? We should at least fix the indentation...

Reply:
Thirded.

Reply:
Ok, I've put CurrentSequence.Scale on the same line, which brings the line to 132 characters. I don't think we should exceed this, as most sites I've seen recommend 120 character lines. I have to say there seems to be a disconnect between the stringency of these line length rules, and the lack of any policy that I can follow, e.g. keep lines to 120 or 130 characters. A lot of this back and forth would have been resolved if there was a clear rule on this. The same can be said for indentation.

On the point of indentation, I am not sure what you mean by "fix" the indentation. Github is not showing it correctly above, in the editor, the parameters fall nicely underneath each other. I don't think it would be a good idea to move the indentation to the left, as then it may look like a separate new line rather than a continuation of the previous line. Though open to ideas on this.

![image](https://user-images.githubusercontent.com/1495872/154843120-03398840-b79b-4f93-8123-a7fc3facb681.png)

Reply:
We have a stylecop rule to not exceed 180 characters I think, other than that we try to keep things on one line where possible.

We usually only indent line breaks by a single tab, which is normally enough to show that it's not a separate new line.